src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -120598,5 +120619,115 @@ public static void Main()
                 options: TestOptions.DebugExe);
             CompileAndVerify(executeComp, expectedOutput: "ran");
         }
+
+        [Fact]
+        [WorkItem(39220, "https://github.com/dotnet/roslyn/issues/39220")]
+        public void GotoMayCauseAnotherAnalysisPass_01()
+        {
+            var source =
+@"#nullable enable
+class Program
+{
+    static void Test(string? s)
+    {
+        if (s == null) return;
+
+        heck:
+        var c = GetC(s);
+        var prop = c.Property;
+        prop.ToString(); // BOOM (after goto)
```

Comment:
>BOOM (after goto) [](start = 28, length = 17)

Consider removing comment since presumably it does not apply.

Reply:
It describes the fact that a null reference exception will occur at runtime.  I'll reword the comment.

---
In reply to: [366648998](https://github.com/dotnet/roslyn/pull/39301#discussion_r366648998) [](ancestors = 366648998)